NetKAN/VesselView-UI.frozen

```diff
@@ -1 +0,0 @@
-VesselView-UI
```

Comment:
_VesselView_ is the main mod. _VesselView-UI-Toolbar_ and _VesselView-UI-RasterPropMonitor_ are both providing the virtual mod **VesselView-UI**. You needed to replace VesselView-UI-Toolbar.frozen and VesselView-UI-RasterPropMonitor.frozen, but VesselView-UI.frozen shouldn't be conflicting, since you don't have a VesselView-UI.netkan